src/main/java/net/snowflake/client/core/IncidentUtil.java

```diff
@@ -206,7 +206,7 @@ private static void writeVmMetrics(JsonGenerator json, VirtualMachineMetrics vm)
         // This function can throw an error with java 11.
         json.writeNumberField("fd_usage", vm.fileDescriptorUsage());
       } catch (Exception e) {
-        logger.info("Error writing fd_usage", e);
+        logger.debug("Error writing fd_usage", e);
```

Comment:
based on the comment above (line 206) I would say that's it's fine to log on debug - it may just fail on java 11 but does not fail anything in process